docs/architecture/ADR-004-interest-and-reserves.md

```diff
@@ -16,18 +16,40 @@ When interest accrues, the sum of all assets owed by all users increases for eac
 
 The mechanism by which interest is calculated, and then split between incentivizing lenders as per [ADR-001](./ADR-001-interest-stream.md) and reserves as defined in this ADR, will follow.
 
-
 ## Decision
 
 Interest on all open borrow positions will be applied every X blocks during `EndBlock`, where X is a governance parameter called `BorrowInterestEpoch`.
 
-Reserves are implemented as a per-token value attached to the `x/leverage` module account.  The module account also holds the lending pool of base assets. As an example, if the module account contains 2000 `uatom` and its `uatom ReserveAmount` is 500, then only 1500 of its `uatom` are available for borrowing or withdrawal.
+Reserves are implemented as a per-token value attached to the `x/leverage` module account. The module account also holds the lending pool of base assets. As an example, if the module account contains 2000 `uatom` and its `uatom ReserveAmount` is 500, then only 1500 of its `uatom` are available for borrowing or withdrawal.
 
 A governance parameter `ReserveFactor` must be kept which specifies the portion of borrow interest that will be funnelled into reserves. This parameter is set _per token_, which allows it to be set differently for riskier assets vs stablecoins, for example.
 
 Both the reserved amount of a given token, and the token:uToken exchange rate, increase when interest is _accrued_, rather then when it is _repaid_.
 
-This timing of reserve increases matches the behavior of the [Compound cToken smart contract](https://github.com/compound-finance/compound-protocol/blob/master/contracts/CToken.sol) we are using as a reference - see around line 410 in function accrueInterest.
+This timing of reserve increases, matches the behavior of the [Compound cToken smart contract](https://github.com/compound-finance/compound-protocol/blob/master/contracts/CToken.sol) we are using as a reference - see around line 410 in function `accrueInterest`.
+
+### Max collateral utilization
+
+We define a _collateral utilization_ as ratio of lended collateral to total amount of collateral per token. Collateral utilization is 0 when there is no borrow of given token collateral. It is equal to 1, when all token collateral is borrowed. Collateral utilization of token X is growing when lenders withdraw their token X collateral or borrowers take a new loan of token X.
```

Comment:
Add some math or specifics here - is this _borrow utilization_ which is `borrowed / (borrowed + module balance - reserves)` or is it _collateral utilization_, which would have to distinguish between lenders with uTokens enabled vs. disabled as collateral?